### PR TITLE
Testing

### LINKED ISSUE #49: Why not output to syslog?
The log are currently written in the `/var/log/nodered/nodered.log` log file.

I wonder if that wouldn't be more convenient to output logs to syslog so we can check everything using the `journalctl` command. What do you think?

### PR DESCRIPTION
**Pull request by tituspijean**
Closes #48 
Closes #49 

## PR Status

- [x] Code finished and ready to be reviewed/tested
- [x] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by tituspijean** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -129,19 +132,11 @@ ynh_script_progression --message="Configuring a systemd service..." --weight=1
...
-
-# Use logrotate to manage application logfile(s)
-ynh_use_logrotate
-
 #=================================================
 # ADVERTISE SERVICE IN ADMIN PANEL
 #=================================================
 
-yunohost service add $app --description="Low-code programming for event-driven applications" --log="/var/log/$app/$app.log"
+yunohost service add $app --description="Low-code programming for event-driven applications" --log_type="systemd"
```
Comment (fflorent) [inline scripts/install:139]: Thanks for taking into account #49!

A little question: you mentioned that this is taken from the `example_ynh`, I wonder what is most compliant regarding the Yunohost standard.

I wonder if:
 - my request is legit and leave it as is;
 - if I should suggest to globalise this change through an issue in `example_ynh` ;
 - my initial request is actually a bad idea;

(mayhaps i scratch a little too much y head for nothing ^^)

**Review by fflorent — APPROVED** [review]
Lgtm

File: scripts/install (hunk 1 above)
Comment (tituspijean) [reply scripts/install:139]: Let's try it, it does not cost much to reverse it if we discover it hinders debuggability. :)

**Pull request merged**